salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume-cleanup

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+import argparse
+import errno
+import os
+import stat
+import sys
+
+
+SPARSE_FILES_DIR = '/var/lib/metalk8s/storage/sparse/'
+LOOP_MAJOR = 7
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser(
+        prog='metalk8s-sparse-volume-cleanup',
+        help='Cleanup MetalK8s sparse loop volumes',
+    )
+
+    parser.add_argument(
```

Comment:
If this is required, why use a flag argument instead of a positional one?

Reply:
Sorry, remnants of the temporary implementation where I had an `action` (`provision` or `cleanup`) as positional. Anyway, will change.